Gemfile

```diff
@@ -2,6 +2,7 @@
 # Bundler will treat runtime dependencies like base dependencies, and
 # development dependencies will be added by default to the :development group.
 gemspec
+gem "sshkey",                         "~>1.8.0",           :require => false
```

Comment:
This should go in the gemspec: https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/manageiq-providers-ibm_cloud.gemspec#L30

Reply:
I was not sure. Let me move it there.